runtimes/eden/src/lib.rs

```diff
@@ -122,6 +119,9 @@ construct_runtime! {
 
 		// DAO
 		DaoReserve: pallet_reserve::<Instance4> = 60,
+
+		// Smart Contracts.
+		Contracts: pallet_contracts::{Pallet, Call, Storage, Event<T>} = 62,
```

Comment:
```suggestion
		Contracts: pallet_contracts = 62,
```
I don't think you need the syntactic sugar anymore.